.storybook/preview-head.html

```diff
@@ -369,7 +369,6 @@
 
 	.tokens__list {
 		flex-grow: 2;
-		margin: 0 40px auto -20px; /* -20px to override storybook default padding */
```

Comment:
do you know what this was for before? how does removing this change things?

Reply:
not sure what it's purpose was currently, but if we leave it in it pulls the category titles off the left of the viewport so i think it makes sense to remove.

<img width="393" alt="Screenshot 2022-07-26 at 12 57 55" src="https://user-images.githubusercontent.com/77005274/181000670-49c41c60-af7e-47f8-822a-0d092be66bb2.png">

Reply:
is this still needed? it will affect every story (i think it may have been for embedding in zeroheight – @SiAdcock?

Reply:
IIUC it's only used in the `palette` story

Reply:
apologies - i didn't notice the classname it's restricted to